moveit2_rolling.repos

```diff
@@ -0,0 +1,49 @@
+repositories:
+  moveit_msgs:
```

Comment:
Just curious.. once rolling is working on all dependencies, we won't really need this file anymore, right? Or is it still required for checking the lastest versions before the corresponding rolling release is out? I'm wondering how changes that affect multiple repos would be synced then.

Reply:
This file is needed because there are dependencies that are not released for rolling yet that are not in the normal repos file.

Reply:
Yes, I already understood so far. The question is just if there is a recommended strategy for syncing changes in multiple repos without breaking Rolling (the release) all the time.